pkg/node/manager.go

```diff
@@ -300,6 +300,7 @@ func DeleteNode(ni Identity, routesTypes RouteType) {
 
 			deleteTunnelMapping(n.IPv4AllocCIDR)
 			deleteTunnelMapping(n.IPv6AllocCIDR)
+			deleteIPRoute(n)
```

Comment:
deleteIPRoute is currently tailored to direct routing rules. We have to reverse `replaceNodeRoute()`